_episodes/08-conflict.md

```diff
@@ -0,0 +1,338 @@
+or get rid of the change entirely.
+Let's replace both so that the file looks like this:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+We removed the conflict on this line
+~~~
+{: .output}
+
+To finish merging,
+we add `mars.txt` to the changes being made by the merge
+and then commit:
+
+~~~
+$ git add mars.txt
+$ git status
+~~~
+{: .bash}
+
+~~~
+On branch master
+All conflicts fixed but you are still merging.
+  (use "git commit" to conclude merge)
+
+Changes to be committed:
+
+	modified:   mars.txt
+
+~~~
+{: .output}
+
+~~~
+$ git commit -m "Merge changes from marsTemp"
+~~~
+{: .bash}
+
+~~~
+[master 2abf2b1] Merge changes from marsTemp
+~~~
+{: .output}
+
+Git keeps track of what we've merged with what,
+so we don't have to fix things by hand again. If
+we make another change to the `marsTemp` branch
+
+~~~
+$ git checkout marsTemp
+$ echo "The polar caps will probably be Yeti's home" >> mars.txt
+$ git add mars.txt
+$ git commit -m "A note about Yeti's home"
+~~~
+{: .bash}
+
+~~~
+[master 34avo82] A note about Yeti's home
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
+And merge that change into master
+
+~~~
+$ git checkout master
+$ git merge marsTemp
+~~~
+
+~~~
+Updating 12687f6..x792csa1
+Fast-forward
+ mars.txt | 1 +
+ 1 file changed, 1 insertions(+), 0 deletions(-)
+~~~
+{: .output}
+
+There is no conflict and our changes are added automatically
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+We removed the conflict on this line
+The polar caps will probably be Yeti's home
+~~~
+{: .output}
+
+We don't need to merge again because Git knows someone has already done that.
+
+Git's ability to resolve conflicts is very useful, but conflict resolution
+costs time and effort, and can introduce errors if conflicts are not resolved
+correctly. If you find yourself resolving a lot of conflicts in a project,
+consider one of these approaches to reducing them:
+
+- Try breaking large files apart into smaller files so that it is less
+  likely that two authors will be working in the same file at the same time
+- Clarify who is responsible for what areas with your collaborators
+- Discuss what order tasks should be carried out in with your collaborators so
+  that tasks that will change the same file won't be worked on at the same time
+
+> ## Create a conflict between branches and resolve it
+>
+> - Create a new branch off of the master branch
+> - Make a change to a file in the master branch
+> - Change to the new branch
+> - Make a change to the same line in the same file
+> - Change back to the master branch
+> - Merge the new branch into the master branch
+> - Address the resulting conflict in the text editor of your choice
```

Comment:
Definitely. This was mostly a copy paste from the original exercise. A good addition